Sources/Public/iOS/LottieAnimationViewBase.swift

```diff
@@ -39,7 +39,11 @@ open class LottieAnimationViewBase: UIView {
 
   var screenScale: CGFloat {
     #if os(iOS) || os(tvOS)
-    UIScreen.main.scale
+    if #available(iOS 13.0, *) {
```

Comment:
We should also check for tvOS 13+

```suggestion
    if #available(iOS 13.0, tvOS 13.0) {
```

Reply:
That's what I missed! Thank you🙂